libraries/net/include/graphene/net/node.hpp

```diff
@@ -247,10 +250,17 @@ namespace graphene { namespace net {
          */
         static std::vector<fc::ip::endpoint> resolve_string_to_ip_endpoints( const std::string& in );
 
-        /**
-         *  Call with true to enable listening for incoming connections
+         /**
+          * listen for incoming connections
```

Comment:
The removal of `enable` made the meaning of the comment totally different (and more confusing).